tests/network_test.cpp

```diff
@@ -1040,16 +1040,16 @@ TEST_F(NetworkTest, node_sync_with_transactions) {
   // To make sure blocks are stored before starting node 2
   taraxa::thisThreadSleepForMilliSeconds(1000);
 
-  EXPECT_EQ(node1->getDagManager()->getNumVerticesInDag().first, 7);
-  EXPECT_EQ(node1->getDagManager()->getNumEdgesInDag().first, 8);
+  EXPECT_GT(node1->getDagManager()->getNumVerticesInDag().first, 6);
+  EXPECT_GT(node1->getDagManager()->getNumEdgesInDag().first, 7);
```

Comment:
This is not correct.
Node1 custom DAG by ourself, we add 6 DAG blocks.
The DAG should be exactly as what we generate. 6 DAG blocks + genesis block = 7 blocks. And base on the DAG we generate, there should have 8 edges. 
The original is correct:
EXPECT_EQ(node1->getDagManager()->getNumVerticesInDag().first, 7);
EXPECT_EQ(node1->getDagManager()->getNumEdgesInDag().first, 8);

Reply:
It is a custom DAG but the transactions from custom DAG sync and then there is a chance that additional blocks are created from these transactions